### PR TITLE
Added full italian language support

### PR DESCRIPTION
**Pull request by Nutpile**
The files in the list below are the only ones i touched so i'd check back on those just to make sure i didn't make a mess
Makefile
src/data.asm
src/flash.asm
src/installer.asm
src/strings.asm

I also tried to keep the string lenghs as similar as possible to the original but italian is just a longer language so sometimes it just wasn't an option

either way i had a lot of fun doing this, see you next time.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -51,28 +53,34 @@ french:
...
 	convbin -k 8xp-compressed -e zx0 -u -n CESIUM -j 8x -i $(BIN_DUTCH) -o $(BIN_DUTCH).compressed.8xp
-	rm -f $(BIN_ENGLISH) $(BIN_FRENCH) $(BIN_DUTCH) $(RELEASE_ZIP)
+	convbin -k 8xp-compressed -e zx0 -u -n CESIUM -j 8x -i $(BIN_DUTCH) -o $(BIN_ITALIAN).compressed.8xp
+	rm -f $(BIN_ENGLISH) $(BIN_FRENCH) $(BIN_DUTCH) $(BIN_ITALIAN)$(RELEASE_ZIP)
 	rm -rf  $(RELEASE_DIR)
 	mkdir -p  $(RELEASE_DIR)
 	mv $(BIN_ENGLISH).compressed.8xp $(RELEASE_DIR)/$(BIN_ENGLISH)
 	mv $(BIN_FRENCH).compressed.8xp $(RELEASE_DIR)/$(BIN_FRENCH)
 	mv $(BIN_DUTCH).compressed.8xp $(RELEASE_DIR)/$(BIN_DUTCH)
+	mv $(BIN_DUTCH).compressed.8xp $(RELEASE_DIR)/$(BIN_ITALIAN)
```
Comment (jacobly0) [inline Makefile:76]:
```suggestion
mv $(BIN_ITALIAN).compressed.8xp $(RELEASE_DIR)/$(BIN_ITALIAN)
```

File: Makefile [hunk 2]
```diff
@@ -51,28 +53,34 @@ french:
...
 	convbin -k 8xp-compressed -e zx0 -u -n CESIUM -j 8x -i $(BIN_DUTCH) -o $(BIN_DUTCH).compressed.8xp
+	convbin -k 8xp-compressed -e zx0 -u -n CESIUM -j 8x -i $(BIN_ITALIAN) -o $(BIN_ITALIAN).compressed.8xp
 
 release: all
 	convbin -k 8xp-compressed -e zx0 -u -n CESIUM -j 8x -i $(BIN_ENGLISH) -o $(BIN_ENGLISH).compressed.8xp
 	convbin -k 8xp-compressed -e zx0 -u -n CESIUM -j 8x -i $(BIN_FRENCH) -o $(BIN_FRENCH).compressed.8xp
 	convbin -k 8xp-compressed -e zx0 -u -n CESIUM -j 8x -i $(BIN_DUTCH) -o $(BIN_DUTCH).compressed.8xp
-	rm -f $(BIN_ENGLISH) $(BIN_FRENCH) $(BIN_DUTCH) $(RELEASE_ZIP)
+	convbin -k 8xp-compressed -e zx0 -u -n CESIUM -j 8x -i $(BIN_DUTCH) -o $(BIN_ITALIAN).compressed.8xp
+	rm -f $(BIN_ENGLISH) $(BIN_FRENCH) $(BIN_DUTCH) $(BIN_ITALIAN)$(RELEASE_ZIP)
```
Comment (jacobly0) [inline Makefile:70]:
```suggestion
	rm -f $(BIN_ENGLISH) $(BIN_FRENCH) $(BIN_DUTCH) $(BIN_ITALIAN) $(RELEASE_ZIP)
```

File: Makefile (hunk 1 above)
Comment (jacobly0) [inline Makefile:76]:
```suggestion
	mv $(BIN_ITALIAN).compressed.8xp $(RELEASE_DIR)/$(BIN_ITALIAN)
```

**Review by jacobly0 — CHANGES_REQUESTED** [review]

**Comment by mateoconlechuga** [comment]
Thanks for translation! Added and cleaned up the makefile to make it easier in the future :)

**Pull request merged**